sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -597,6 +619,107 @@ private[hive] class HiveMetastoreCatalog(val client: HiveClient, hive: HiveConte
     }
   }
 
+  private def convertToOrcRelation(metastoreRelation: MetastoreRelation): LogicalRelation = {
```

Comment:
I have done a refac to have a single method named `convertToLogicalRelation` instead of separate ones for parquet and orc